app/controllers/calls_controller.rb

```diff
@@ -45,9 +52,9 @@ def connection_gather_prompt
         action: calls_connection_gather_url(connection_id: active_connection.id),
         timeout: 15
       ) do |gather|
-        gather.Say 'How did the senator respond? Press 1 for positively, press 2 for negatively'
+        r.Play AudioFileFetcher.file_for_key('user_response')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.